zebra/rt_netlink.c

```diff
@@ -3421,7 +3421,14 @@ ssize_t netlink_nexthop_msg_encode(uint16_t cmd,
 											 nh,
 											 buflen);
 						break;
-					default:
+					case ZEBRA_SEG6_LOCAL_ACTION_END_DX2:
```

Comment:
and this is exactly why I want action to be the appropriate enum.  The compiler would tell us that we are not properly handling all the cases.